git_flow.rst

```diff
@@ -0,0 +1,189 @@
+.. -*- mode: rst -*-
+
+.. role:: bash(code)
+   :language: bash
+
+Recommended GitHub workflow
```

Comment:
Thanks for writing this, since obviously I never got around to it. This document is a lot like what other sites call the "how to contribute" page, which is how I was envisioning writing it when it was still on _my_ TODO list.  Do you object to me doing some moderately heavy editing of the prose to make it more in line with that idea?  In other words, making less like "here's how to work with git" and more like "if you want to use expyfun, here's how we do it"?